frontend/src/container/NewWidget/LeftContainer/QuerySection/index.tsx

```diff
@@ -125,10 +115,10 @@ function QuerySection({
 			return;
 		}
 
-		setQueryCategory(parseInt(`${qCategory}`, 10));
+		setQueryCategory(qCategory as EQueryType);
 		const newLocalQuery = {
 			...cloneDeep(query),
-			queryType: parseInt(`${qCategory}`, 10),
+			queryType: qCategory as EQueryType,
```

Comment:
maybe we can check `EQueryType` same way we are using `isTraceFilterEnum`

Reply:
qCategory is used as a string argument for the tabs onChange callback